blender-2.5/exporter/osg/osgdata.py

```diff
@@ -157,6 +157,27 @@ def findArmatureObjectForTrack(track):
     return None
 
 
+def unselect_all_objects():
+    for obj in bpy.context.selected_objects:
+        obj.select = False
+
+
+def make_dupliverts_real(scene):
+    ''' Duplicates vertex instances and makes them real'''
+    unselect_all_objects()
```

Comment:
Ho, right. I didn't think about this case. We need to save the selection and restore it after the preprocess.